extensions/wikia/InsightsV2/scripts/LoopNotification.js

```diff
@@ -0,0 +1,136 @@
+/*
+ * Script for adding BannerNotification that encourages user to continue working
+ * on issues reported by Insights feature
+ */
+
+/* global require */
+require(
+	[
+		'jquery',
+		'BannerNotification',
+		'wikia.querystring',
+		'wikia.window',
+		'ext.wikia.Insights.LoopNotificationTracking'
+	],
+	function ($, BannerNotification, Querystring, window, loopNotificationTracking)
+{
+	'use strict';
+
+	$(function () {
+		var qs = new Querystring(),
+			insights = qs.getVal('insights', null),
+			isVE = qs.getVal('veaction', null),
+			isFixed = false,
+			isEdit = false,
+			notification,
+			notificationType,
+			// Functions
+			initNotification,
+			showNotification,
+			getMessageType,
+			getParent,
+			addInsightsFlowToEditButtons;
+
+		showNotification = function(response) {
+			if (response) {
+				var msgType,
+					$parent = getParent();
+
+				isFixed = response.isFixed;
+				msgType = getMessageType();
+
+				if (notificationType !== response.notificationType) {
+					notificationType = response.notificationType;
+
+					if (notificationType === 'notfixed') {
+						addInsightsFlowToEditButtons();
+					}
+
+					if (notification) {
+						notification.hide();
+					}
+					notification = new BannerNotification(response.html, msgType, $parent).show();
+				}
+
+				loopNotificationTracking.setParams(isEdit, isFixed, notificationType);
+
+				if (notificationType === 'fixed') {
+					$('#InsightsNextPageButton').focus();
+				} else if (notificationType === 'notfixed') {
+					$('#InsightsEditPageButton').focus();
+				}
+
+				if (isFixed) {
+					loopNotificationTracking.successTrack();
+				}
+			}
+		};
+
+		getMessageType = function() {
+			if (isEdit || !isFixed) {
+				return 'warn';
+			} else {
```

Comment:
I think it always looks cleaner if you remove the unnecessary else block when there is a return in the if (same in the method below), i.e.
```javascript
getMessageType = function() {
	if (isEdit || !isFixed) {
		return 'warn';
	}
	return 'confirm';
}
```